lib/beautiful/xresources.lua

```diff
@@ -70,13 +70,21 @@ local function get_screen(s)
     return s and screen[s]
 end
 
---- Get global or per-screen DPI value falling back to xrdb.
--- @tparam[opt] integer|screen s The screen.
+--- Get global or per-screen DPI value, falling back to xrdb.
+-- @screen[opt] s The screen.
 -- @treturn number DPI value.
 function xresources.get_dpi(s)
-    s = get_screen(s)
-    if dpi_per_screen[s] then
-        return dpi_per_screen[s]
+    if s then
```

Comment:
Uhm, this means `nil` is now a valid argument? Why? I know that this might make things easier for some callers, but doesn't that also mean it becomes more easy to break per-screen-DPI?